cmd/flarectl/misc.go

```diff
@@ -147,7 +147,11 @@ func pageRules(c *cli.Context) {
 			case float64:
 				s = fmt.Sprintf("%s: %.f", cloudflare.PageRuleActions[a.ID], v)
 			case map[string]interface{}:
-				s = fmt.Sprintf("%s: %.f - %s", cloudflare.PageRuleActions[a.ID], v["status_code"], v["url"])
+				if a.ID == "minify" {
+					s = fmt.Sprintf("%s: [js:%s html:%s css:%s]", cloudflare.PageRuleActions[a.ID], v["js"], v["html"], v["css"])
+				} else {
```

Comment:
This is a little hap-hazardous as we're using an `if`/`else` and checking for the `minify` ID as if it's a special case amongst another group of common values (which it's not as it's a 50/50 between the two options). I think we might be better off either:

- Defining a custom formatter that is reusable for all `map[string]interface{}`. Perhaps a field to value output? Example, `[status_code: 301 url: https://example.com]` and `[js: true html:true css: true]` type of thing.
- Explicitly calling out the two conditions in their own `if`s to avoid any accidental classifications.

I think the first one is a more appropriate action to ensure we get some consistency but I'm not too fussed. I just foresee some accidents if we leave it as is.